examples/k8s-bookinfo.yaml

```diff
@@ -57,150 +54,138 @@ spec:
           value: "true"
         - name: A8_CONTROLLER_URL
           value: http://$(CONTROLLER_SERVICE_HOST):$(CONTROLLER_SERVICE_PORT)
-        - name: A8_REGISTRY_URL
-          value: http://$(REGISTRY_SERVICE_HOST):$(REGISTRY_SERVICE_PORT)
+        - name: A8_REGISTRY_BACKEND
+          value: kubernetes
         - name: A8_SERVICE
-          value: gateway:none
+          value: gateway
         - name: A8_LOGSTASH_SERVER
```

Comment:
This was added for a specific reason. Nginx access logs were complaining that the version info was missing.